src/utils/translations/pt.ts

```diff
@@ -0,0 +1,94 @@
+const pt = {
+  // Home Tab
+  Home: "Início",
+  "Period in": "Período em",
+  "no info": "desconhecido",
```

Comment:
As I understand, the `desconhecido` in Portuguese means `unknown` in English, not `no info`